lib/generators/solidus_stripe/install/install_generator.rb

```diff
@@ -3,26 +3,28 @@
 module SolidusStripe
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      class_option :migrate, type: :boolean, default: true
       class_option :auto_run_migrations, type: :boolean, default: false
-      class_option :auto_run_seeds, type: :boolean, default: false
+
+      def add_javascripts
```

Comment:
should we also remove these requires?

Reply:
I was thinking the same... but maybe it's better to conditionally load the files if present (and have `solidus_dev_support` do it). I think it makes sense to keep this kind of boilerplate. @aldesantis what do you think?

Reply:
I think this issue is related https://github.com/solidusio/solidus_dev_support/issues/45

Reply:
We'll get to it! 🙏